modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js

```diff
@@ -69,7 +70,7 @@ const ClayColorPickerWithState = ({
 		<ClayColorPicker
 			aria-describedby={describedBy}
 			aria-invalid={invalid}
-			aria-label="color"
+			aria-label={label}
```

Comment:
@rijema, have you tested it with different languages locally? I understand it works for field having options, but I'm little unsure about the other fields without options.